src/Microsoft.Health.Dicom.Core/Exceptions/UnauthorizedDicomActionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Exceptions
+{
+    public class UnauthorizedDicomActionException : DicomServerException
+    {
+        public UnauthorizedDicomActionException()
```

Comment:
Should we add more details on what was expected and what was passed, so it is easier to troubleshoot for our customers

Reply:
@brandonpollett, will this be addressed?

Reply:
I think it would be a good idea to log the exceptions, but I don't think we should make the error message to the end user any more verbose.